src/app.jsx

```diff
@@ -324,12 +324,24 @@ class Application extends React.Component {
         cockpit.script("echo $XDG_RUNTIME_DIR")
                 .done(xrd => {
                     sessionStorage.setItem('XDG_RUNTIME_DIR', xrd.trim());
-                    this.init(false);
+                    if (!utils.isRootUser()) {
```

Comment:
You can't (reliably) call isRootUser() that way, as it returns a promise. Also, I don't quite like the setState() call below, this feels like duplication.

How about simplifying this and checking root-ness in the shell command above? Like

    [ `id -u` -eq 0 ] || echo $XDG_RUNTIME_DIR

and handling the empty case in the `.done` section. This will also nicely cover the case when the runtime dir is not set due to other reasons, in which case podman won't work anyway.